src/sql/workbench/services/connection/browser/connectionDialogService.ts

```diff
@@ -495,6 +504,12 @@ export class ConnectionDialogService implements IConnectionDialogService {
 		recentConnections.forEach(conn => conn.dispose());
 	}
 
+	public launchChangePasswordDialog(profile: IConnectionProfile, params: INewConnectionParams): void {
```

Comment:
Please add doc for public API

Reply:
Done so in the IConnectionDialogService interface definition (located in src/sql/workbench/services/connection/common/connectionDialogService.ts), but I'll add the comment here as well